packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -65,6 +66,8 @@ const customModuleRegexes = bpkReactScriptsConfig.babelIncludePrefixes
   : [];
 
 const optInCssModules = bpkReactScriptsConfig.cssModules === false;
+const optimizeCssOptions = bpkReactScriptsConfig.optimizeCssOptions ? bpkReactScriptsConfig.optimizeCssOptions : {}
```

Comment:
I see. I guess the whole premise of create-react-app (and thus backpack-react-scripts) is to hide the configuration away from consumers. If there is something that a consumer / team deems necessary to include then why wouldn't we wanna include that for everyone? Im tempted to remove it.